backend/src/main/java/com/mcmasterbaja/analyzer/ConstantAdderAnalyzer.java

```diff
@@ -0,0 +1,67 @@
+package com.mcmasterbaja.analyzer;
+
+import com.opencsv.CSVReader;
+import com.opencsv.ICSVWriter;
+import com.opencsv.exceptions.CsvValidationException;
+import java.io.IOException;
+
+public class ConstantAdderAnalyzer extends Analyzer {
+  private final double a;
+  private final double b;
+  private final double c;
+  private final double d;
+
+  public ConstantAdderAnalyzer(
+      String[] inputFiles,
+      String[] inputColumns,
+      String[] outputFiles,
+      double a,
+      double b,
+      double c,
+      double d) {
+    super(inputFiles, inputColumns, outputFiles);
+    this.a = a;
+    this.b = b;
+    this.c = c;
+    this.d = d;
+  }
+
+  public void analyze() throws IOException, CsvValidationException {
+
+    System.out.println(
+        "Add a constant value to a file named"
+            + super.inputFiles[0]
+            + " to make "
+            + super.outputFiles[0]);
+    CSVReader r = getReader(super.inputFiles[0]);
+    ICSVWriter w = getWriter(super.outputFiles[0]);
```

Comment:
Nitpicky but every other analyzer calls these `reader` and `writer` respectively, may be nice to stay consistent

Reply:
sure will change